website/docs/Troubleshooting/01-MinimizingNoise.md

```diff
@@ -25,9 +25,9 @@ Ensure that your electrodes are connected securely (especially your reference)!
 
 Every so often, an error will occur with the wireless communication between your OpenBCI Dongle and board. If you've followed all of the steps above, and the data that you are seeing in the GUI interface is still illegible, try the following:
 
-1. Power down your Cyton board and unplug your USB Dongle.
-2. Then, plug back in your USB Dongle and power up your Cyton board in that order. 
-3. Last, try restarting the system by pressing the `START SYSTEM` button again.
+1. Power down your board and unplug your USB Dongle.
+2. Plug back in your USB Dongle and power up your board in that order. 
+3. Restart the GUI and start a new session.
 
 #### Check Cyton - GUI connection
```

Comment:
Under this heading "Check Cyton", we have some more button names that are lowercase and inside quotes. Can we capitalize and make bold?